apps/masterbots.ai/components/routes/chat/chat-list.tsx

```
@@ -78,95 +79,167 @@ export function ChatList({
+    <div className='flex items-center w-full h-20 opacity-65 gap-4'>
+      {(() => {
+        switch (activeTool?.toolName) {
+          case 'webSearch':
+            return (
+              <>
+                <GlobeIcon className="size-6 animate-bounce relative top-2" />
+                <p className="leading-none flex flex-col gap-1">
+                  <span>
+                    Searching on the web{' '}
+                    {Array.from({ length: 3 }).map((_, index) => (
+                      <span key={index} className="animate-pulse rounded-full text-4xl h-0.5 leading-none" style={{ animationDelay: `${index * 100}ms` }}>.</span>
```

Biome

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)